conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/mantis/MantisFilter.scala

```diff
@@ -0,0 +1,56 @@
+package tech.cryptonomic.conseil.api.routes.platform.data.mantis
+
+import tech.cryptonomic.conseil.api.routes.platform.data.ApiFilter._
+import tech.cryptonomic.conseil.common.generic.chain.DataTypes
+import tech.cryptonomic.conseil.common.generic.chain.DataTypes.{OperationType, Predicate, Query}
+
+/**
+  * Represents a query filter submitted to the Conseil API.
+  *
+  * @param limit                  How many records to return
+  * @param blockIds               Block IDs
+  * @param transactionHashes      Transaction IDs
+  * @param sortBy                 Database column name to sort by
+  * @param order                  Sort items ascending or descending
+  */
+case class MantisFilter(
```

Comment:
Again not for this PR, but I'm wondering why not a `RawBlockchainFilter`... i mean, Mantis, Bitcoin and Ethereum filters are exactly the same... I almost sure this is a general observation across the code.